core/src/main/java/org/eclipse/hono/config/ProtocolAdapterProperties.java

```diff
@@ -140,7 +140,7 @@ public void setDefaultsEnabled(final boolean flag) {
      *
      * @return The timeout value in milliseconds.
      */
-    public final long getEventLoopBlockedCheckTimeout() {
+    public long getEventLoopBlockedCheckTimeout() {
```

Comment:
@chapitos : I had another look into the class, and sorry, I think it should be the other way round:
both get and set should be final (matching all the other getters and setters in the property-classes).
The only exception is the `defaultsEnabled` property, but shouldn't be part of this PR.
It usually just doesn't make sense to overwrite a simple getter and setter.

Can you please make both of them final and push a new commit?
Thanks!

Reply:
Done.